genai-perf/README.md

```diff
@@ -556,6 +556,31 @@ The number of tokens in each prefix prompt. This value is only used if
 being concatenated, the number of tokens in the final prompt may be off by one.
 (default: `100`)
 
+##### `--image-width-mean <int>`
+
+The mean width of images when generating synthetic image data.
+(default: `100`)
+
+##### `--image-width-stddev <int>`
+
+The standard deviation of width of images when generating synthetic image data.
+(default: `0`)
+
+##### `--image-height-mean <int>`
+
+The mean height of images when generating synthetic image data.
+(default: `100`)
+
+##### `--image-height-stddev <int>`
+
+The standard deviation of height of images when generating synthetic image data.
+(default: `0`)
+
+##### `--image-format <str>`
+
+The compression format of the images. If format is not selected,
+format of generated image is selected at random.
```

Comment:
There's not a default compression format?

Reply:
Yes, when image format is not specified, synthetic generator randomly selects the format: https://github.com/triton-inference-server/perf_analyzer/blob/9e682f4550cb9635b2254fba10c4baa47cd27f39/genai-perf/genai_perf/inputs/retrievers/synthetic_image_generator.py#L57

Reply:
Interesting. I think it'd be helpful to give JPEG/PNG as examples, as I didn't immediately know that's what this meant. But users can always use genai-perf profile --help to see that, so I don't think this is imperative.